src/client.rs

```diff
@@ -40,27 +40,32 @@ impl Client {
     ) -> Self {
         Client {
             client,
+            field_manager: field_manager.clone(),
             post_params: PostParams {
                 field_manager: field_manager.clone(),
                 ..PostParams::default()
             },
             patch_params: PatchParams {
-                field_manager: field_manager.clone(),
-                ..PatchParams::default()
-            },
-            // TODO: According to https://kubernetes.io/docs/reference/using-api/server-side-apply/#using-server-side-apply-in-a-controller we should always force conflicts in controllers.
-            //  we currently consider this a workaround until we can properly implement patching
-            //  (see https://github.com/stackabletech/operator-rs/issues/113)
-            force_patch_params: PatchParams {
                 field_manager,
-                force: true,
                 ..PatchParams::default()
             },
             delete_params: DeleteParams::default(),
             default_namespace,
         }
     }
 
+    /// Server-side apply requires a `field_manager` that uniquely identifies a single usage site,
+    /// since it will revert changes that are owned by the `field_manager` but not part of the Apply request.
+    fn apply_patch_params(&self, context: impl Display) -> PatchParams {
```

Comment:
Not really a fan of the name `context`, since it conflicts with `Context`. `scope` has similar problems (`kube` already uses `scope` to talk about namespaced vs cluster-global). `field_manager_scope`, maybe?

Reply:
`field_manager_scope` would work I guess, maybe something with `reference` was the only other thing I came up with ..